### PR TITLE
Support class constants for services' names

### PR DESCRIPTION
**Pull request by hectorj**
Hi!

With the newest versions of Symfony, it has become a common practice to use fully qualified class names as services' ids.

So I added support for using class constants like `Foo::class` as argument to `get()`

### CODE REVIEW

**Comment by hectorj** [comment]
I have some code style issues + a nullable return type to remove

**Comment by lookyman** [comment]
Thank you, I'll have a look at it later today. Meanwhile, please check out the failing Travis build.

EDIT: You were 3 seconds faster. :)

File: src/ServiceMap.php [hunk 1]
```diff
@@ -43,9 +50,35 @@ public function __construct(string $containerXml)
...
+		}
+
+		return null;
+	}
+
+	/**
+	 * @param mixed $node
+	 * @return null|string
+	 */
+	public static function getServiceIdFromNode($node)
```
Comment (hectorj) [inline src/ServiceMap.php:72]: I'm not a fan of having this here... but I needed a way to get the ID there : https://github.com/hectorj/phpstan-symfony/blob/ba8dc8d950a5da6117172f1e231fe55b75f34715/src/Rules/ContainerInterfaceUnknownServiceRule.php#L38

I'm open to suggestions.
Don't hesitate to edit the PR if you want to refactor.

**Comment by lookyman** [comment]
I'm going to pick this up again and get it implemented. Sorry I was MIA..

**Comment by lookyman** [comment]
Implemented as 57f8fcb. Thank you!

**Pull request closed without merging**